contracts/stellar-multicall/src/types.rs

```diff
@@ -0,0 +1,9 @@
+use soroban_sdk::{contracttype, Address, Symbol, Val, Vec};
+
+#[contracttype]
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct MulticallData {
+    pub contract_address: Address,
+    pub function: Symbol,
+    pub args: Vec<Val>,
+}
```

Comment:
```suggestion
pub struct FunctionCall {
    pub contract: Address,
    pub function: Symbol,
    pub args: Vec<Val>,
}
```
this is a single function call, not a multicall. And avoid using generic terms like data, params for names